package.json

```diff
@@ -34,10 +35,10 @@
     "reactify": "1.0.0",
     "restify": "4.3.1",
     "sdc-clients": "10.2.1",
-    "ufds": "1.3.0",
     "superagent": "0.21.0",
     "through": "2.3.4",
     "trace-event": "1.3.0",
+    "ufds": "1.1.3",
```

Comment:
why roll it back??

Reply:
I used npm i --save and this command makes the alphabetical sorting in the dependencies.

Reply:
this happens after npm i -- save and rebase